src/olympia/addons/models.py

```diff
@@ -954,6 +954,12 @@ def current_version(self):
             pass
         return None
 
+    @amo.cached_property(writable=True)
+    def latest_unlisted_version(self):
+        """Shortcut property for Addon.find_latest_version(
+        channel=RELEASE_CHANNEL_UNLISTED)."""
```

Comment:
Is this just for the serializer? Should there be a `latest_listed_version` shortcut property?